test/lib/processors/normalize-headers-processor.test.js

```diff
@@ -0,0 +1,30 @@
+const libingester = require('../../../');
+
+const cheerio = require('cheerio-advanced-selectors').wrap(require('cheerio'));
+const fs = require('fs-extra');
+const path = require('path');
+
+async function readHtml (name) {
+    return fs.readFile(path.join(__dirname, name), 'utf8');
+}
+
+let $body;
+beforeAll(async () => {
+    // FIXME needed to initialize config and logger
+    // eslint-disable-next-line no-new
+    new libingester.WebIngester();
+    const html = await readHtml('normalize-headers-processor.in.html');
+    $body = cheerio.load(html).root();
+});
+
+test('can process headers', async () => {
+    const out = await readHtml('normalize-headers-processor.out-1.html');
+    const result = libingester.processors.processNormalizeHeaders($body);
+    expect(result.$body.html()).toBe(out);
+});
+
+test('does not pass h6', async () => {
+    const out = await readHtml('normalize-headers-processor.out-2.html');
+    const result = libingester.processors.processNormalizeHeaders($body, { fromIndex: 5 });
+    expect(result.$body.html()).toBe(out);
+});
```

Comment:
What you normally do is to use [describe](https://jestjs.io/docs/en/api#describename-fn) to group tests if they are related, they are going to be grouped afterwards in the output.